README.md

```diff
@@ -173,10 +173,12 @@ To build with [CMake](https://cmake.org/):
 
 ```bash
 $ mkdir -p out/cmake ; cd out/cmake ; cmake -DBUILD_TESTING=ON ../..
-$ make all test
-# Or manually:
-$ ./uv_run_tests    # shared library build
-$ ./uv_run_tests_a  # static library build
+$ make all test                       # on Unix
+$ cmake --build . ; ctest -C Debug    # on Windows
```

Comment:
cmake + ctest works on both so we might as well simplify this and drop the `make all test` line.

Reply:
Sounds good! I also will add `--output-on-failure` option to `ctest` because `ctest` would not show any test log whether tests pass or failure by default.